### PR TITLE
improved card design

### PR DESCRIPTION
**Pull request by suveshmoza**
I have updated the card layout and also made them to have same height.

### CODE REVIEW

**Comment by RedFox0x20** [comment]
All looks good, much nicer card design that works well with the changes in #61 

The only issue is the `Health Education Workshop` card where on a desktop display, the button text overflows slightly at the bottom.

**Comment by audreyfeldroy** [comment]
I love your work here @suveshmoza and the code looks very good. 

About the Health Education Workshop card, I noticed that before. I think I can just shorten the wording as a quick fix 🙂

**Comment by audreyfeldroy** [comment]
Thanks so much @RedFox0x20 for thoughtfully reviewing both this and #68 🙏

File: components/HealthWorkshopCard.js [hunk 1]
```diff
@@ -36,14 +39,16 @@ function HealthWorkshopCard() {
           {'Also, your donation will go toward health-themed mini gift bags to motivate people to attend a workshop, equipment such as trainer simulaids for edema and breast disease, venue costs, and to hire public health educators to prepare presentations.'}
         </p>
         <p>
+          <br/>
           <form onSubmit={handleSubmit}>
-            <button className="btn btn-primary" type="submit">
-              {'Help Create 1 Health Workshop for $5000'}
+            <button className="btn btn-primary " type="submit">
+              {'Help Create 1 Health Workshop for $5000 '}
```
Comment (audreyfeldroy) [inline components/HealthWorkshopCard.js:45]:
```suggestion
              {'Create 1 Health Workshop for $5000 '}
```

**Review by audreyfeldroy — APPROVED** [review]
All looks good. The suggested change I made is technically out of scope of this PR...just adding it here to address @RedFox0x20's comment before I forget.

**Comment by audreyfeldroy** [comment]
Just wanted to say that I truly appreciate how thoughtful you were about choosing such nice photos. And that the design you created is really beautiful. Thank you @suveshmoza.

**Pull request merged**